widgets/actionsMenu.widget.js

```diff
@@ -119,7 +119,7 @@ class myWidget extends baseWidget() {
   }
 
   getWidget () {
-    return this.grid.gridObj.set(...this.grid.gridLayout, this.blessed.list, {
```

Comment:
This was causing the actions menu to always be drawn behind the main container view, it was just not noticeable unless you resized the terminal window to specific screen sizes. 
<img width="2151" alt="image" src="https://github.com/lirantal/dockly/assets/26180666/58c76774-1239-4d36-8640-86ae6c106398">
You can see in this screenshot the menu poking out to the right of the container status panel ^